scene/2d/animated_sprite.cpp

```diff
@@ -74,6 +78,18 @@ bool AnimatedSprite::_edit_use_rect() const {
 	}
 	return t.is_valid();
 }
+
+void AnimatedSprite::get_argument_options(const StringName &p_function, int p_idx, List<String> *r_options) const {
+	if (p_idx == 0 && p_function == "play" && frames.is_valid()) {
+		const String quote_style = EDITOR_GET("text_editor/completion/use_single_quotes") ? "'" : "\"";
+		List<StringName> al;
```

Comment:
I'm not very familiar with this but I noticed some of the other places in 3.x that do this, use this code:
```
#ifdef TOOLS_ENABLED
	const String quote_style = EDITOR_GET("text_editor/completion/use_single_quotes") ? "'" : "\"";
#else
	const String quote_style = "\"";
#endif
```
Is this necessary or does the EDITOR_GET work ok in non-tools builds?

Reply:
Ah, this whole `get_argument_options()` function is inside `TOOLS_ENABLED`, so it's not necessary to use the `#ifdef`.

I'm not sure if it's best practice. But I'm going to change it to use the `#ifdef` approach for the sake of consistency.